.github/workflows/xcop.yml

```diff
@@ -1,15 +0,0 @@
----
-name: xcop
```

Comment:
@advasileva We fixed problem with `xcop`. https://github.com/g4s8/xcop-action/issues/4

Reply:
@volodya-lombrozo It's still redundant (because it is checked in the maven build), but let it be